src/dialects/abstract/connection-manager.ts

```diff
@@ -190,7 +191,7 @@ export class AbstractConnectionManager<TConnection extends Connection = Connecti
   }
 
   async #initDatabaseVersion() {
-    if (this.sequelize.options.databaseVersion !== 0) {
+    if (!isNil(this.sequelize.options.databaseVersion)) {
```

Comment:
`isNil` just checks if it's `== null`. Do we need that? I think we could just write `databaseVersion !== null`, right?
Also; what does `isNil(0)` produce?

Reply:
So `isNil` checks for both the cases:

```js
isNil(null);    // true
isNil(0);    // true

isNil(NaN);    // false
```

Reply:
Are you sure? `isNil(0)` seems to return false in http://jsfiddle.net/tomohidex/uxy1t4b1/

Reply:
Ok weird I got it confused with `isNil(void 0)` I'll update this to what you suggested! Thanks a lot @WikiRik, I can't believe I used to use this!

Reply:
isNil (in the case of lodash) means undefined or null, but we've pretty much banned it because nowadays null-or-undefined is usually called "nullish", not nil

I personally prefer `== null`, it's the only scenario where I'm ok with `==` instead of `===`